recipes/paratext/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: paratext
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/wiseio/paratext.git
+  git_rev: 2f2824eb8c8f1d060c1105531bc7ab9c44b55246
```

Comment:
We prefer tarballs rather than tags. Can you change this to:

```yaml
{% set commit = "2f2824eb8c8f1d060c1105531bc7ab9c44b55246" %}

source:
  fn: paratext-{{ version }}.tar.gz
  url: https://github.com/wiseio/paratext/archive/{{ commit }}.tar.gz
  sha256: d24368e104d9d44e763e61d6db680ce95439b67596b5f0c411c8eba6abec05c5
```